images/win/scripts/Installers/Install-Runner.ps1

```diff
@@ -0,0 +1,4 @@
+$runnerDownloadUrl = Get-GitHubPackageDownloadUrl -RepoOwner "Actions" -RepoName "runner" -BinaryName "actions-runner-linux-x64" -Version "latest" -UrlFilter "*{BinaryName}-{Version}.zip"
+$fileName = runnerDownloadUrl.Substring($zip.LastIndexOf("/") + 1)
+New-Item -Path "C:\ProgramData\runner" -ItemType Directory
+$kotlinInstallerPath = Start-DownloadWithRetry -Url $runnerDownloadUrl -Name $fileName -DownloadPath "C:\ProgramData\runner"
```

Comment:
lets introduce a pester test in order to make sure that the target directory exists and not empty